pykechain/client.py

```diff
@@ -798,16 +831,43 @@ def properties(
     def property(self, *args, **kwargs) -> 'AnyProperty':  # noqa: F
         """Retrieve single KE-chain Property.
 
-        Uses the same interface as the :func:`properties` method but returns only a single pykechain :class:
-        `models.Property` instance.
+        Uses the same interface as the :func:`properties` method but returns only a single
+        pykechain :class:`models.Property` instance.
 
-        If additional `keyword=value` arguments are provided, these are added to the request parameters. Please
-        refer to the documentation of the KE-chain API for additional query parameters.
+        If additional `keyword=value` arguments are provided, these are added to the request
+        parameters. Please refer to the documentation of the KE-chain API for additional query
+        parameters.
+
+        When only the `pk` is provided as an input for the part search, the detail route will
+        be called.
 
         :return: a single :class:`models.Property`
         :raises NotFoundError: When no `Property` is found
         :raises MultipleFoundError: When more than a single `Property` is found
         """
+        property_id = None
```

Comment:
Indeed a bit of duplicated code, can we find a smart way to move it under a function?

If you don't think it's worth the effor then just close it. Looks good, otherwise!